itest/tapd_harness.go

```diff
@@ -200,10 +200,11 @@ func newTapdHarness(t *testing.T, ht *harnessTest, cfg tapdConfig,
 		TLSPath:      cfg.LndNode.Cfg.TLSCertPath,
 	}
 
-	// Ensure valid proof from tapd nodes will be accepted, and proofs will
-	// be queryable by other tapd nodes. This applies to federation syncing
-	// as well as RPC insert and query.
-	tapCfg.Universe.PublicAccess = true
+	// Configure the universe server to ensure that valid proofs from tapd
+	// nodes will be accepted, and proofs will be queryable by other tapd
+	// nodes.
+	tapCfg.Universe.PublicAccess =
```

Comment:
micro-nit: treat the cast as a function call and wrap/format the lines accordingly.